virtool/uploads/utils.py

```diff
@@ -76,3 +79,16 @@ async def naive_writer(
             size += len(chunk)
 
     return size
+
+
+async def get_upload_path(config: Config, name_on_disk: str) -> str:
```

Comment:
This returns a `pathlib.Path` object.
```suggestion
async def get_upload_path(config: Config, name_on_disk: str) -> Path:
```